### DISCUSSION TITLE
don't repeat the question

### DISCUSSION THREAD

**Discussion by sazearte** [nomic-ai/pygpt4all#81] [General]
don't repeat the question
when I start a chat, I have in output my question plus the answer of the model, is it possible to hide the question I ask him?
I have this example:

```
msg="what is kubernetes?"
output = model.generate(msg, n_predict=55, n_threads=8)
print(output)
```

when I do print(output)
I don't want to display my question: "what is kubernetes ?", I want to see only its answer, how to do ?

**Comment by absadiki** [comment]
Hi @sazearte,

Please update the package to the latest version and run it as follows:

```python
for token in model.generate(msg):
    print(token, end='', flush=True)
```

Let me know if that soled the issue ?

**Comment by sazearte** [comment]
it's work, thanks
